blackbox/docs/best_practice/migrating_from_mysql.txt

```diff
@@ -148,6 +148,24 @@ Finally use the ``COPY FROM`` statement to import your newly generated json file
   Read :ref:`efficient_data_import` for more information how to import huge datasets into Crate.
 
 
+3rd Party Tool: csvkit
+======================
+
+The Python tools provided by `csvkit`_ allow you to directly insert CSV data
+into Crate via SQLAlchemy, using Crate’s native driver to create the table,
```

Comment:
The ~~Python~~ tools provided by `csvkit`_ allow you to directly insert CSV data into Crate via SQLAlchemy, using the [Crate Python client](https://pypi.python.org/pypi/crate) to create the table, ...